src/apps/vpn/connectionbenchmark/connectionbenchmark.cpp

```diff
@@ -218,11 +222,14 @@ void ConnectionBenchmark::handleStabilityChange() {
   if (m_state == StateInitial || m_state == StateReady) {
     return;
   }
+  logger.debug() << "Handle stability change";
+  checkStability();
+}
 
+void ConnectionBenchmark::checkStability() {
```

Comment:
Could this be renamed to something different? Current naming of this function confuses me, I would expect a return value (such as `ConnectionHealth::ConnectionStability` or even `bool`) because it is checking something but it doesn't return anything.

Reply:
100% great call out, thank you. What do you think about `showErrorIfNoSignal`? It's kinda clunky, open to suggestions or talking through options.

Reply:
`showErrorIfNoSignal()` sounds great, thank you!